curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f78193bb0510709b169b8.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 671f78193bb0510709b169b8
+title: Task 107
+challengeType: 19
+dashedName: task-107
+---
+
+<!-- READING -->
+
+# --instructions--
+
+Read the following text and answer the question below.
+
+# --questions--
+
+## --text--
+
+This is a prologue to the comic on PyCon.
```

Comment:
Comic?

Reply:
Nielda's review proposes a change to solve this. Thanks for noticing it.